hibike/setup.sh

```diff
@@ -9,6 +9,13 @@ cp utils/flash /etc/bash_completion.d/flash.sh
 echo "Setup Tab-complete"
 echo "Please restart your terminals"
 
+echo "Update Arduino library: First command line argument as file path to .tar (or .tar.xz)"
+echo "Update to Arduino 1.8.1"
+if [ $# -eq 1 ] && [ $(grep -c -i "arduino" <<< $1) != 0 ] && [ $(grep -c ".tar" <<< $1) != 0 ]; then
```

Comment:
Nikita caught this. 
http://stackoverflow.com/questions/16045139/redirector-in-ubuntu

oh oops my regex didn't format correctly. 
`.*arduino.*\.tar\.xz`

grep uses regex to look for things